aries_cloudagent/DefaultMessageDelivery/__init__.py

```diff
@@ -3,16 +3,21 @@
 
 from ..config.injection_context import InjectionContext
 from ..core.event_bus import EventBus
-from collections import OrderedDict
-
+from ..core.transport_events import OutboundMessageEvent
+from . import OutboundMessageRouter
 
 LOGGER = logging.getLogger(__name__)
-topics: OrderedDict = {}  #TODO: add topics and listeners
+topics = {
+    OutboundMessageEvent.topic_re : OutboundMessageRouter.outbound_message_event_listener
+    }
 
 
 async def setup(context: InjectionContext):
     ''' subscribe listeners to topics'''
-    event_bus = context.inject(EventBus, required=False)
-    if event_bus:
+    try:
+        event_bus = context.inject(EventBus, required=False)
         for topic, listener in topics:
             event_bus.subscribe(topic, listener)
+    except Exception:
+        LOGGER.exception("Unable to register Default Message Delivery plugin")
```

Comment:
It could be a good idea to catch the exception to be printed in the logger and propagate it later on?

Reply:
What are we expecting an exception for exactly? Catching this does not seem like a good idea, especially given how general the exception is.

Reply:
In other places, we use the event bus we have if statements that test if the event bus exists. It is more pythonic to use try-catch instead of guard-if statements, which makes for faster code. I will just remove the try-catch in favor of not knowing what exception to expect.